### ISSUE TITLE
Feature request: Increase band limit

### ISSUE THREAD

**Issue by d3m3vilurr** [strijar/x6100_gui#4]
current band option is limited [32](https://github.com/strijar/x6100_gui/blob/main/src/bands.c#L19).
can you increase the value to 64 or 128?

I want to update/modify the SQLite database to see the more detailed band plans

**Comment by strijar** [comment]
Yes it is possible. I plan to implement a band editor soon, so this code will be reworked.
